models/classes/runner/config/QtiRunnerConfig.php

```diff
@@ -147,6 +147,14 @@ protected function buildConfig()
             ];
         }
 
+        // Send a unique value so the frontend knows if the test has been reopened
+        // in another tab (that allows it to not modify the local storage from two
+        // different tabs)
+        //
+        $config['runNumberId'] = floor(microtime(true) * 100);
+
+        // @todo If the session is being restored, force restoreTimerFromClient to be true
+
```

Comment:
@js the FE does not need this value to be passed in the payload of the config endpoint, right?

It it's not needed, I'll remove changes in this file

Reply:
Correct, this is not needed anymore. The client-side implementation is taking care if this. See https://github.com/oat-sa/tao-test-runner-qti-fe/pull/522/files#diff-943b82aebd513800263e6f86920439485c546a4f7897a8bc65036d5ce0936f61R57

Reply:
On it

Reply:
3f527a61c6f881f0ff063427bd30720517f6aedf